boards: arm: doc: Delete Segger RTT and Add BLE 5.0 in nrf52_vbluno52

iotmember opened the pull request:
Delete Segger RTT and Add BLE 5.0 version in nrf52_vbluno52 document

This patch fix problems in nrf52_vbluno52 document:
+ Delete `Segger RTT` from supported features
+ Add `5.0` version to Bluetooth Low Energy feature

Signed-off-by: Nam Do <robotden@gmail.com>

iotvietmember commented:
@dbkinder @galak I hope that this commit could be reviewed soon. Thank you very much!

nashif commented:
just fix your commit message please

iotvietmember commented:
@nashif could you please give me more detail information? Thank you

nashif commented:
look here:

https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/2571/3/tests

iotvietmember commented:
@nashif I have just fixed the problem. Many thanks

nashif commented:
Commit message still has problems. Please install gitlint and run it on your commits as described here: https://github.com/zephyrproject-rtos/zephyr/wiki/Contribution-Guide

dbkinder reviewed (dismissed):
+1 for content change

iotmember committed:
boards: arm: doc: Fix problems in nrf52_vbluno52 document

This patch fix problems in nrf52_vbluno52 document:
+ Delete `Segger RTT` from supported features
+ Add `5.0` version to Bluetooth Low Energy feature

Signed-off-by: Nam Do <robotden@gmail.com>

iotvietmember commented:
@nashif @dbkinder @galak I have just fixed the problems. Thank you for your support!
